docs/devices.ipynb

```diff
@@ -0,0 +1,357 @@
+{
```

Comment:
<p>maybe add that the following only throws one exception?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/5395/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>If this is saying that a circuit with multiple exception-throwing issues (eg unsupported qubit and gate simultaneously), will only throw one exception, it might be good to show this in an example. </p>

Reply:
I think just showing the single exceptions on their own here is fine, unless we feel very strongly here ?